charts/flannel-0.25.7/values.yaml

```diff
@@ -0,0 +1,70 @@
+---
+global:
+  imagePullSecrets:
+# - name: "a-secret-name"
+
+# The IPv4 cidr pool to create on startup if none exists. Pod IPs will be
+# chosen from this range.
+podCidr: "10.244.0.0/16"
+podCidrv6: ""
+
+flannel:
+  # kube-flannel image
+  image:
+    repository: container-registry.oracle.com/olcne/flannel
```

Comment:
this should just be olcne/flannel

Reply:
Is it something new behaviour added recently ? I just made it similar to flannel-0.22.3